package.json

```diff
@@ -6,6 +6,7 @@
     "test": "pnpm -r --workspace-concurrency=1 test",
     "dashboard:start": "cd packages/db-dashboard && pnpm run dev",
     "dashboard:build": "cd packages/db-dashboard && pnpm run build",
+    "ra-data-rest": "pnpm -F @platformatic/db-ra-data-rest",
```

Comment:
What’s -F?

Reply:
It's the same of "yarn workspace", it's a shortcut to run commands into a package.

So you can write from the root:

`pnpm ra-data-rest <whatever script inside that package.json>`